tools/clang/test/HLSL/vector-syntax.hlsl

```diff
@@ -91,10 +91,10 @@ void vector_out_of_bounds() {
 void vector_unsigned() {
    unsigned int4 intvector;
    unsigned min16int4 min16vector;
-   unsigned int64_t3 int64vector;
+   unsigned int64_t3 int64vector; /* */
    unsigned uint3 uintvector;
    unsigned min16uint4 min16uintvector;
-   unsigned uint64_t2 int64uintvector;
+   unsigned uint64_t2 int64uintvector;                      /* */
```

Comment:
Did tools do something weird here?

Reply:
Yeap... not sure why it generated blank comment